pkg/services/ngalert/api/tooling/definitions/provisioning_templates.go

```diff
@@ -16,9 +24,70 @@ package definitions
 //       200: MessageTemplate
 //       404: NotFound
 
+// swagger:route PUT /api/provisioning/templates/{ID} provisioning RoutePutTemplate
```

Comment:
I think we should use `{name}` instead of `{ID}`, wdyt?

Reply:
yes, that is 100% better